<launch>
    <arg name="task_type" doc="Which task has to run : [welding, cleaning]" />
    <arg name="task_planner_notify_end"     default="true"/>
    <arg name="simu" doc="Simulation or real robot : [ON, OFF]" default="ON" />

    <arg name="toy_waypoints" doc="Publish hand written waypoints to help debugging / testing : [true, false]" default="true"/>
    <arg name="waypoints_filename" doc="File containing waypoints, check wp5-mam-tasks/config folder" default="$(find wp5_mam_tasks)/config/waypoints.yaml"/>

    <arg name="robot_type" doc="Which robot to use : [ur5, ur10e]" default="ur10e"/>
    <param name="robot_type" value="$(arg robot_type)" />

    <group ns="$(arg robot_type)">
        <node pkg="wp5_mam_tasks"
              type="wp5_task_node"
              name="wp5_task_node"
              output="screen">
            <param name="task_type" value="$(arg task_type)" />
            <param name="task_planner_notify_end" value="$(arg task_planner_notify_end)"/>
            <param name="toy_waypoints" value="$(arg toy_waypoints)" />
            <param name="waypoints_filename" value="$(arg waypoints_filename)" />
        </node>
    </group>
</launch>